app/assets/stylesheets/tinymce/skins/alchemy/skin.min.css.scss

```diff
@@ -638,10 +640,10 @@ body .mce-abs-layout-item,.mce-abs-end {
     text-align: center;
     overflow: visible;
     -webkit-appearance: none;
-    @include border-radius(inherit);
+    border-radius: inherit;
 
     &:focus {
-      @include box-shadow(none);
+      box-shadow: none;
       border: 0 none;
       background: none;
     }
```

Comment:
We need to add an override for 

```sass
.mce-btn button:active { border: 0 none; }
```

Otherwise we will have duplicated borders when pressing the button:

![alchemy cms - all you can eat 2016-07-05 23-16-29](https://cloud.githubusercontent.com/assets/42868/16600321/d07624ae-4306-11e6-8202-44e411b73f03.jpg)